search/tests/mock_search_engine.py

```diff
@@ -318,7 +318,7 @@ def __init__(self, index=None):
         super(MockSearchEngine, self).__init__(index)
         MockSearchEngine.load_index(self.index_name)
 
-    def index(self, sources):  # pylint: disable=arguments-differ
+    def index(self, sources, **kwargs):
```

Comment:
Should these`kwargs` be used somewhere?

Reply:
They don't need to be here. The main problem is that this method has a different signature from the `ElasticsearchBackend`'s version of `index` so the tests are failing when trying to pass through the `timeout` kwarg, even though the actual indexing works fine.